### DISCUSSION TITLE
Visualize a configurable amount of log lines around search results

### DISCUSSION THREAD

**Discussion by andreapicaro** [SigNoz/signoz#4065] [Q&A]
Hi, I'm totally new to SigNoz and just looking into features for evaluation.
I was wondering if it is possible to select a configurable amount of log lines immediately before and after a search hit.
Let's say I find a log line by keyword search, I want to visualize that line with immediate surroundings to understand the context. 
Is that possible?

Thanks

**Comment by pranay01** [comment]
Yes @andreapicaro. You can use `View in context` in the logs search results

Check this short video - https://www.loom.com/share/ea95ca24dd604ba0ba25a3e3a6b21adf?sid=9f3456fd-47e6-446b-99ae-1e68b336a5e4
